compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -901,4 +900,23 @@ object messages {
     val msg = hl"trying to define package with same name as `$existing`"
     val explanation = ""
   }
+
+  case class ExistentialTypesNoLongerSupported()(implicit ctx: Context) extends Message(34) {
+    val kind = "Syntax"
+    val msg = "Existential types are no longer supported - use a wildcard or dependent type instead"
+    val explanation =
+      hl"""| The use of existential types is no longer supported and a wildcard or dependent type should be used instead
+           |
+           | For example:
+           |
+           | Instead of using ${"forSome"} to specify a type variable
+           |
+           | ${"List[T forSome { type T }]"}
+           |
+           | Try using a wildcard type variable
+           |
+           | ${"List[_]"}
+           |"""
```

Comment:
Throughout this explanation, there's no need for there to be a padding of whitespace after the `|`-signs. None of the other messages should be doing this.